twistr/space_race.go

```diff
@@ -0,0 +1,67 @@
+package twistr
+
+import "fmt"
+
+type SRId int
```

Comment:
SRId is a little terse for a package-wide name.

Maybe change "SR" everywhere here to "Space"?